graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/fetchers/admin/DeploySchemaResponseDto.java

```diff
@@ -0,0 +1,64 @@
+package io.stargate.graphql.schema.schemafirst.fetchers.admin;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import graphql.GraphqlErrorHelper;
+import graphql.schema.DataFetchingEnvironment;
+import io.stargate.graphql.schema.schemafirst.migration.MigrationQuery;
+import io.stargate.graphql.schema.schemafirst.processor.ProcessingLogType;
+import io.stargate.graphql.schema.schemafirst.processor.ProcessingMessage;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class DeploySchemaResponseDto {
+
+  private UUID version;
+  private List<ProcessingMessage<ProcessingLogType>> logs;
+  private List<MigrationQuery> cqlChanges;
+
+  public void setVersion(UUID version) {
+    this.version = version;
+  }
+
+  public UUID getVersion() {
+    return version;
+  }
+
+  public void setLogs(List<ProcessingMessage<ProcessingLogType>> logs) {
+    this.logs = logs;
+  }
+
+  public List<Map<String, Object>> getLogs(DataFetchingEnvironment environment) {
```

Comment:
It doesn't look like this or `getCqlChanges` below are used. Is that right or is Intellij confused?

Reply:
graphql-java calls them at runtime, it matches the GraphQL field to the name of the property. We have to use POJOs for fields that can be parameterized, that doesn't work with the map approach.

And in fact I wished we had relied on POJOs more from the start, it's a cleaner approach.